### ISSUE TITLE
Tensorflow 2.0 version issue

### ISSUE THREAD

**Issue by arwhirang** [wsjeon92/FP2VEC#2]
Hello,

I installed TensorFlow 2 version on my computer. To run the FP2VEC code, I changed the code only the part that covers compatibility with TensorFlow 1 version without fixing other parts. However, when I ran the code, the loss becomes nan and the FP2VEC can't learn anything. I hope you can convert the code into TF2 version.

I attached my slightly changed code here.
[Ten2fp2vec_original.zip](https://github.com/wsjeon92/FP2VEC/files/3994389/Ten2fp2vec_original.zip)

**Comment by wsjeon92** [comment]
Thank you for your opinion. We checked the code and found that there was a problem in the function of tf.nn.dropout(). The argument "Rate" should be set to rate = 1 - keep_prob, where the "keep_prob" is TF 1.0 argument. Please check the details in the below links.
https://www.tensorflow.org/api_docs/python/tf/nn/dropout?version=stable
https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/nn/dropout

If you change the rate as 1-keep_prob in the function "conv_model" in class "model", it runs properly. 
I uploaded the compatible version of code for TF 2.0 in the "Tox21" directory. Please check and let us know if there is a problem or question.

Thank you.

Sincerely, Woosung Jeon
